corrector.rb

```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name(name)
+    name.capitalize!
+    name[0...10] if name.length > 10
+  end
```

Comment:
- [ ] Nice job for implementing this method :+1: . From the requirements of this project, this method should raise a` NotImplementedError`.